ui/admin/tests/acceptance/workers/create-test.js

```diff
@@ -38,6 +39,28 @@ module('Acceptance | workers | create', function (hooks) {
     assert.strictEqual(getWorkersCount(), workersCount + 1);
   });
 
+  test('cluster id input field is visible for `hcp` binary', async function (assert) {
+    assert.expect(1);
+    const config = getOwner(this).resolveRegistration('config:environment');
+    config.featureFlags['byow-pki-hcp-cluster-id'] = true;
+    config.featureFlags['byow-pki-upstream'] = false;
+    await visit(newWorkerURL);
+    assert
```

Comment:
Can you also assert the field in each test doesn't exist?

Reply:
@ZedLi what do you think of the `doesNotIncludeText` assertion?

Reply:
I think instead of looking for a specific `nth-child` that for something doesn't exist you should do a `find` for the label that shouldn't exist and confirm it's not found